docker/server-pytorch/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'com.bmuschko.docker-remote-api'
+}
+
+evaluationDependsOn ':docker-server'
+
+def dockerContext = project.layout.buildDirectory.dir('context')
+
+def prepareDocker = project.tasks.register('prepareDocker', Sync) {
```

Comment:
wouldn't it make more sense to put these contents in docker/server itself? we don't have a plugin/extension which can only happen once per project, these are just plain tasks, so we can stack them up (or make them generic and declare several sets of them). This removes the need for evaluationDependsOn and project().tasks.findByName() calls, since the tasks are right there and can be strongly referenced

Reply:
I've been tending to prefer one docker image per project; and maybe there is a way in the future where we can use the docker images as artifacts (like you've suggested with the application.tar), and we can do away w/ the eval depends on. Even though it's a bit more setup, I think it's easier to understand as a self contained unit (something that JJ or others can grok more easily having it separated).

Reply:
Done